modules/widgets/social-icons/social-icons-admin.js

```diff
@@ -0,0 +1,111 @@
+( function( $ ) {
+	var timeout = null;
+
+	// Make the list of items sortable.
+	function initWidget( widget ) {
+		widget.find( '.jetpack-social-icons-widget-list' ).sortable( {
+			items: '> .jetpack-social-icons-widget-item',
+			handle: '.handle',
+			cursor: 'move',
+			placeholder: 'jetpack-social-icons-widget-item ui-state-placeholder',
+			containment: widget,
+			forcePlaceholderSize: true,
+			update: function() {
+				livePreviewUpdate( $( this ).parents( '.form' ).find( '.widget-control-save' ) );
+			}
+		} );
+	}
+
+	// Live preview update.
+	function livePreviewUpdate( button ) {
+		if ( ! $( 'body' ).hasClass( 'wp-customizer' ) || ! button.length  ) {
```

Comment:
May be more efficient to use `document.body` or -- if jQuery is needed -- `$( document.body )` instead of passing in a string element selector?  Unsure if jQuery has some optimization behind the scenes for this already.